Week-1/Session-02/Assignments/Intermediate/HTML_Table.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML Table</title>
+</head>
+
+<body>
+    <table border="double">
+        <!-- <thead>
+            <td colspan="3">
+                <h1>1</h1>
+            </td>
+        </thead> -->
+        <tbody>
+            <tr>
+                <td colspan="3">
+                    <h1>1</h1>
+                </td>
+            </tr>
+            <tr>
+                <td rowspan="2">
+                    <h1>2</h1>
+                </td>
+                <td>
+                    <h1>3</h1>
+                </td>
+                <td>
+                    <h1>4</h1>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <h1>5</h1>
+                </td>
+                <td>
+                    <h1>6</h1>
+                </td>
+            </tr>
+        </tbody>
+    </table>
+
+    <ol>
+        <li>Desktop</li>
+        <ul>
```

Comment:
Style should be circle as per question
<ul style="list-style-type:circle;">

Reply:
As I am seeing in the document nowhere is mentioned about the Styling.

![image](https://user-images.githubusercontent.com/36344687/232842874-58c6f9a7-2af6-40ad-be9e-6a59dec94fdf.png)

Reply:
no but look at the unordered list here, default is of type disc which is solid black filled bullet point
but here its circle.

Reply:
Correct but when I opened my HTML file in the browser the ul list comes with a circle only

![image](https://user-images.githubusercontent.com/36344687/232868873-a56d2bc0-bdfe-461f-9b0d-dacbb9f55ad6.png)

Reply:
turns out default style type for ul inside ol is circle, not disc.